src/main/java/edu/cornell/kfs/sys/CUKFSConstants.java

```diff
@@ -16,6 +16,8 @@
     public static final String RULE_CODE_CA = "CA";
     public static final String DELIMITER = ".";
     public static final String DATE_FORMAT_YYYY = "YYYY";
+    public static final String FILE_EXTENSION_DELIMITER = ".";
```

Comment:
Why is another constant needed? Couldn't DELIMITER which is already present be used?